data/beta/field/copyright.yaml

```diff
@@ -3,4 +3,4 @@ datatype: text
 field: copyright
 phase: beta
 register: ''
-text: Copyright for the data in the register.
+text: The owner or copyright that protects the data in a register.
```

Comment:
I don't think this field has been used yet in a register. I think it is confusing to say "The owner". I assume this field is to record the name of the copyright licence under which the data is released.

Better to say:
"Name of the copyright licence agreement under which data in a register is released."